x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecation_logs/fix_deprecation_logs/fix_logs_step.test.tsx

```diff
@@ -35,13 +38,13 @@ const getLoggingResponse = (toggle: boolean): DeprecationLoggingStatus => ({
   isDeprecationLoggingEnabled: toggle,
 });
 
-describe('Overview - Fix deprecation logs step', () => {
-  let testBed: OverviewTestBed;
+describe('ES deprecation logs', () => {
```

Comment:
I don't think we need this test file. We could move all the tests directly in `es_deprecation_logs.test` (they have the same `describe`). WDYT?